lib/reducers/create-otp-reducer.js

```diff
@@ -732,15 +733,40 @@ function createOtpReducer (config) {
         })
 
       case 'NEARBY_STOPS_RESPONSE':
+        const {parentStopId, stops} = action.payload
         const stopLookup = {}
-        action.payload.stops.forEach(s => {
+        stops.forEach(s => {
           stopLookup[s.id] = s
         })
+
+        const stopIds = stops.map(stop => stop.id).filter(id => parentStopId !== id)
+
+        if (!parentStopId) {
+          return update(state, {
+            location: {
+              nearbyStops: { $set: stopIds }
+            },
+            transitIndex: { stops: { $merge: stopLookup } }
+          })
+        } else {
+          // We need to remove the parent stop from the stop lookup to avoid overwriting it and
+          // losing the child stops field we're adding here.
+          delete stopLookup[parentStopId]
+          return update(state, {
+            transitIndex: { stops: {
+              // We'll keep all of the main child stop objects in the stops lookup (to match how the state currently looks)
+              $merge: stopLookup, // going to include the parentStop
+              // For the parent stop, we want to add the nearby stops as a new child stops field
+              [parentStopId]: {$merge: {childStops: stopIds}}
```

Comment:
Can we not have `parent*` and `child*` here? I would instead prefer `focusStop` and `nearbyStops`.